integration-tests/api/__tests__/store/cart/cart.js

```diff
@@ -931,6 +948,11 @@ describe("/store/carts", () => {
     beforeEach(async () => {
       await cartSeeder(dbConnection)
       await swapSeeder(dbConnection)
+
+      await simpleSalesChannelFactory(dbConnection, {
```

Comment:
suggestion: It seams that we always expect the store to have a default SS. should we update the model to change the default ss id to not be nullable? it would avoid unexpected behaviour no?

Reply:
I think that would make the introduction of Sales Channels a breaking change

Reply:
Yeah probably it something to do here 😅 but it would make sense in another ticket if we do not support having a store without a default ss. Something to check later

Reply:
Yeah, makes sense!